### ISSUE TITLE
GitHub repo description to mention "performance testing"

### ISSUE THREAD

**Issue by peter-kehl** [minio/warp#71]
Hi,

Thank you for MinIO and Warp.
Could you update the GitHub repo description, to include

Otherwise it takes people longer to find it. For example: they search for "test", they find https://github.com/minio/s3-benchmark (archived), and only from there they get to Warp = 2 more unnecessary hops. And since the archived repos mention "test," one may not think there would be another performance test repository that wouldn't have "test" in the description.

**Comment by klauspost** [comment]
Definitely. We are looking to include stricter tests in #72 after which it definitely will make sense.

Right now consistency checks are mostly incidental and we don't do strict verification on the returned data, so until that is in place I would hesitate to put 'test' in the description.

Let's keep this issue open until we reach a stage where we can.
